packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testSourceCodeWhenClassAndMethodExist.st

```diff
@@ -0,0 +1,15 @@
+as yet unclassified
+testSourceCodeWhenClassAndMethodExist
+
+	| compiledMethod tree source |
+	
+	compiledMethod :=  (BlockClosure>>#ensure:).
+		
+	tree := IPTTreeModelMock new.
+	tree currentSelection: MessageTallyMock new.
+	
+	tree currentSelection method: compiledMethod.
+	
+	source := compiledMethod methodReference source.
```

Comment:
Could `source` be set to an explicit value to improve the readability and robustness of the test?